apps/web/src/app/api/stream/sandbox/route.ts

```diff
@@ -0,0 +1,46 @@
+import Mux from '@mux/mux-node'
+import { NextResponse } from 'next/server'
+
+import { createClient } from '@supabase/supabase-js'
+
+const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL!
+const supabaseServiceKey = process.env.SUPABASE_SERVICE_ROLE_KEY!
+
+const supabase = createClient(supabaseUrl, supabaseServiceKey)
+
+const mux = new Mux({
+  tokenId: process.env.MUX_TOKEN_ID,
+  tokenSecret: process.env.MUX_TOKEN_SECRET
+})
+
+export async function POST(request: Request) {
+  // TODO: UseG apiKey
+  // const apiKey = request.headers.get('X-API-Key')
+  const { sandboxId } = await request.json()
+  console.log('sandboxId', sandboxId)
+
+  // if (!apiKey) {
+  //   return NextResponse.json({ error: 'Missing X-API-Key' }, { status: 400 })
+  // }
```

Comment:
There's no security?

Reply:
Also, why are we using api routes, our backend is in Go

Reply:
Discussed this @ValentaTomas. We felt like this is a better place for the implementation given it's right next to the actual page rendering the stream.